changelog_entry.yaml

```
@@ -1,4 +1,4 @@
 - bump: minor
   changes:
     fixed:
-    - Montana EITC.
+    - Montana EITC exclude married separate.
```

No need to describe program features.
```suggestion
    - Montana EITC.
```